server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementIterator.java

```diff
@@ -217,10 +218,16 @@ private static boolean shouldReturnDueToSplit(final TabletMetadata tm,
         .collect(Collectors.summarizingLong(Long::longValue)).getSum() > splitThreshold;
   }
 
-  private static boolean shouldReturnDueToLocation(final TabletMetadata tm,
+  private boolean shouldReturnDueToLocation(final TabletMetadata tm,
       final Set<TableId> onlineTables, final Set<TServerInstance> current, final boolean debug) {
+
+    if (migrations.contains(tm.getExtent())) {
+      return true;
+    }
+
     // is the table supposed to be online or offline?
-    final boolean shouldBeOnline = onlineTables.contains(tm.getTableId());
+    final boolean shouldBeOnline =
+        onlineTables.contains(tm.getTableId()) && tm.getOperationId() == null;
```

Comment:
You should be able to modify TabletManagerIteratorIT to insert an OPID column into the tablet metadata and check that the Tablet gets returned by the iterator.

Reply:
done in e2f4500